filebeat/docs/fields.asciidoc

```diff
@@ -64,270 +64,183 @@ Contains fields for the Apache2 HTTPD access logs.
 --
 type: keyword
 
-Client IP address.
+Client IP address or hostname.
 
 
 --
 
-*`apache2.access.user_name`*::
+*`apache2.access.body_sent.bytes`*::
 +
 --
-type: keyword
+type: long
 
-The user name used when basic authentication is used.
+format: bytes
+
+The number of bytes of the server response body.
 
 
 --
 
-*`apache2.access.method`*::
+*`user.name`*::
 +
 --
-type: keyword
-
-example: GET
-
-The request HTTP method.
-
+type: alias
```

Comment:
Is this what will appear in the documentation on aliases? We should probably take the description of the original field or something like this.

Not for this PR in any case as this file is generated.

Reply:
Documentation on aliases is being worked on generically in #9269, don't worry. I agree this looks terrible ;-)